lib/containers/__tests__/gene-tests.js

```diff
@@ -5,6 +5,9 @@ import 'react-native'
 import React from 'react'
 import renderer from 'react-test-renderer'
 
+var refineCallbackPromise = () => Promise.resolve()
+jest.mock('../../native_modules/refine_callback.js', () => { return { triggerRefine: () => refineCallbackPromise() }})
+
```

Comment:
replaces the function which normally [triggers the native code](https://github.com/artsy/emission/blob/master/lib/native_modules/refine_callback.js#L8) with our own instantly resolved promise